hwilib/devices/trezorlib/transport/webusb.py

```diff
@@ -105,19 +108,16 @@ def __init__(
     def get_path(self) -> str:
         return "%s:%s" % (self.PATH_PREFIX, dev_to_str(self.device))
 
-    def get_usb_vendor_id(self) -> int:
-        return self.device.getVendorID()
-
     @classmethod
-    def enumerate(cls) -> Iterable["WebUsbTransport"]:
+    def enumerate(cls, usb_reset=False, usb_ids: Optional[Set[Tuple[int, int]]] = TREZORS) -> Iterable["WebUsbTransport"]:
         if cls.context is None:
             cls.context = usb1.USBContext()
             cls.context.open()
             atexit.register(cls.context.close)
         devices = []
         for dev in cls.context.getDeviceIterator(skip_on_error=True):
             usb_id = (dev.getVendorID(), dev.getProductID())
-            if usb_id not in TREZORS:
+            if usb_ids is not None and usb_id not in usb_ids:
```

Comment:
using `usb_ids=None` this way will let you enumerate **all** vendor-class USB devices present on the system. Some of them might not be user accessible. This crashes your enumeration on my machine.

Reply:
I've pushed a change to drop that.